README.rst

```diff
@@ -40,7 +40,8 @@ The required dependencies to use the software are:
 * SciPy >= 0.7
 * Scikit-learn >= 0.12.1
 * Nibabel >= 1.1.0.
-
+* Six
+=======
```

Comment:
Not too sure where this === comes from but I think it should be removed unless there is a missing title in the rst.